genesis-tools/genesis-populate/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "genesis-populate"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+[dependencies]
+borsh = "0.2.2"
```

Comment:
This is not the latest borsh version

Reply:
Fixing in the upcoming PR.

Reply:
From rust book this is quite confusing:

> Cargo has ensured that every crate in the workspace using the rand crate will be using the same version. Using the same version of rand across the workspace saves space because we won’t have multiple copies and ensures that the crates in the workspace will be compatible with each other.

0.2.2 and 0.2.5 probably both resolve to 0.2.5 in cargo lock. what if they are incompatible?
We have some dependencies in workspace cargo that duplicate to specific package, which one counts?

Reply:
It is confusing, yes. Check out cargo lock, it uses 0.2.2.